src/style/components/asset/link-preview-asset.less

```diff
@@ -32,7 +32,8 @@
 }
 
 .link-preview-image-container {
-  width: @link-preview-height;
+  min-width: @link-preview-height;
+  height: @link-preview-height;
```

Comment:
let's change the variable to rem, the reason height wasn't fixed was for larger font sizes.

80px would be 5rem, wdyt?

Reply:
Oh that's why we had this strange : `width: @...-height` :D

Reply:
Using rem sounds like a nice solution 👍